test/kafka_test.go

```diff
@@ -483,8 +483,53 @@ func (s *CLITestSuite) TestKafkaQuota() {
 	}
 }
 
+func (s *CLITestSuite) TestKafkaConsumer() {
+	test := CLITest{
+		login:   "cloud",
+		args:    "kafka consumer list --consumer-group consumer-group-1 --cluster lkc-1234",
+		fixture: "kafka/consumer/list.golden",
+	}
+
+	s.runIntegrationTest(test)
+
+	test = CLITest{
+		login:   "onprem",
+		args:    fmt.Sprintf("kafka consumer list --consumer-group consumer-group-1 --url %s", s.TestBackend.GetKafkaRestUrl()),
```

Comment:
Do we need to pass `--url` if the login is "onprem"?

Reply:
Yes and no. This url is the rest proxy endpoint and not the mds url, so we still need to provide it after logging in unless we set the `CONFLUENT_REST_URL` environment variable.

A did a quick survey though and it seems that most of our Kafka onprem tests set the env variable instead of using the `--url` flag, so I've switched it over just for consistency.

Reply:
I think there's a ticket for it, but I wonder if in the future, we can infer the Kafka REST URL from `confluent kafka cluster use` or `--cluster` on the on-prem side.